packages/components/src/components/file-uploader/_file-uploader.scss

```diff
@@ -112,7 +112,7 @@
     grid-template-columns: 1fr auto;
     grid-auto-rows: auto;
     align-items: center;
-    min-height: $carbon--spacing-07;
+    min-height: $carbon--spacing-08;
```

Comment:
want to double check if this change is across the board or for non drag-and-drop uploader only based on https://github.com/carbon-design-system/carbon/issues/2288#issuecomment-492421590

Reply:
@laurenmrice any thoughts? This is the part that changes the uploaded file from 32px -> 40px, so we just want to confirm if only the non-drag and drop variants should have the 40px uploaded file size!